assets/section-footer.css

```diff
@@ -457,22 +458,80 @@ noscript .localization-selector.link {
   }
 }
 
+.footer-block-image {
+  display: flex;
+}
+
+.footer-block-image.left {
+  justify-content: flex-start;
+}
+
+.footer-block-image.center {
+  justify-content: center;
+}
+
+.footer-block-image.right {
+  justify-content: flex-end;
+}
+
 @media screen and (max-width: 749px) {
-  .footer-block-image {
-    text-align: center;
+  .footer-block-image,
+  .footer-block-image.left,
+  .footer-block-image.center,
+  .footer-block-image.right {
+    justify-content: center;
   }
 }
```

Comment:
This seem a bit opposite to our usual approach where we tend to have default values on mobile and overwrites in media queries for tablet/desktop, no ?

Reply:
Good point. That's how it was originally designed so I didn't change, but I can refactor this in the follow up PR 👀